quilt_lite/quilt_lite/graphql.py

```diff
@@ -0,0 +1,282 @@
+import datetime
+import functools
+import re
+import typing as T
+
+import ariadne
+import importlib_resources
+import quilt3
+
+
+# TODO: load schema from a public shared folder
+with importlib_resources.path("quilt_lite", "schema.graphql") as schema_path:
+    type_defs = ariadne.load_schema_from_path(str(schema_path))
+
+datetime_scalar = ariadne.ScalarType("Datetime")
+
+@datetime_scalar.serializer
+def serialize_datetime(value):
+    return value.isoformat()
+
+
+query_type = ariadne.QueryType()
+
+
+# Buckets
+@query_type.field("bucketConfigs")
+def query_bucket_configs(*_):
+    return []
+
+@query_type.field("bucketConfig")
+def query_bucket_config(*_, name: str):
+    return None
+
+
+def append_path(base: str, child: str):
+    if not base: return child
+    return base + "/" + child
+
+class PackageDirWrapper:
+    def __init__(self, path: str, subpkg: quilt3.Package):
+        self.path = path
+        self._subpkg = subpkg
+
+    @functools.cached_property
+    def metadata(self):
+        return self._subpkg._meta
```

Comment:
This looks to me like overusing of `cached_property` since call of this function is quite cheap.

Consider
```suggestion
    @property
    def metadata(self):
        return self._subpkg._meta
```
or
```suggestion
    metadata = property(operator.attrgetter('_subpkg._meta'))
```